Resources/Prototypes/_Pirate/Entities/Structures/Doors/Airlocks/access.yml

```
@@ -0,0 +1,120 @@
+- type: entity
+  parent: AirlockInterdyne
+  id: AirlockInterdyneLocked
+  suffix: Interdyne, Locked
+  components:
+  - type: ContainerFill
+    containers:
+      board: [ DoorElectronicsIP ]
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)